Lib/dataclasses.py

```diff
@@ -198,10 +198,13 @@ def __repr__(self):
 
 class _InitVarMeta(type):
     def __getitem__(self, params):
-        return self
+        return InitVar(params)
 
 class InitVar(metaclass=_InitVarMeta):
-    pass
+    __slots__ = ('type', )
```

Comment:
We should probably have a nicer `__repr__` here.

Currently you get:
```
>>> from dataclasses import InitVar
>>> InitVar[int]
<dataclasses.InitVar object at 0x00A041C0>
```

I'd suggest something like:
```
    def __repr__(self):
        return f'dataclasses.InitVar[{self.type}]'
```
for starters, although `typing.ClassVar` does something more sophisticated in place of `{self.type}`. Maybe someday when we import `typing` from `dataclasses` we can switch to using their helper function. It's probably worth adding a comment to that effect, too.

Reply:
Using `f'dataclasses.InitVar[{self.type}]'` would look like:

```python
>>> from dataclasses import InitVar
>>> InitVar[int]
dataclasses.InitVar[<class 'int'>]
```

So I added `f'dataclasses.InitVar[{self.type.__name__}]'` instead. Should I write a test for this.

Reply:
If that's what gives you the equivalent of:
```
>>> import typing
>>> typing.ClassVar[int]
typing.ClassVar[int]
```
Then I'm all for it.

Yes, we'll need tests for this. Nothing complex, `InitVar[int]` should be good enough.

Reply:
Sorry for the delay, I added a really simple test:

`self.assertEqual(InitVar[int].type, int)`

Would you using `__repr__` and `@dataclass`?

Reply:
I apologize for taking forever to get back to you on this.

Could you test the `repr` of an `InitVar`? Something like (untested) `repr(InitVar[int]) == 'dataclasses.InitVar[int]'`?

I promise I'll commit this as soon as you add this change.